packages/commonwealth/server/router.ts

```diff
@@ -273,26 +276,40 @@ function setupRouter(
     '/createThread',
     passport.authenticate('jwt', { session: false }),
     databaseValidationService.validateAuthor,
+    databaseValidationService.validateChain,
     createThread.bind(this, models, tokenBalanceCache, ruleCache, banCache)
   );
   router.put(
     '/editThread',
     passport.authenticate('jwt', { session: false }),
     databaseValidationService.validateAuthor,
+    databaseValidationService.validateChain,
     editThread.bind(this, models, banCache)
   );
+  router.put('/editThread', passport.authenticate('jwt', { session: false }), [
```

Comment:
Why is this duplicated?

Reply:
deleting, I think this happened from a weird merge I did